server/common/src/main/java/org/eclipse/lsp/cobol/common/mapping/ExtendedDocument.java

```diff
@@ -182,6 +183,13 @@ public boolean isLineEmptyBetweenColumns(int lineno, int start, int end) {
     return true;
   }
 
+  private RangeInc updateRangeDueToChanges(RangeInc range) {
+    if (isDirty()) {
+      range = new RangeInc(updatePositionDueToChanges(range.getStart()), updatePositionDueToChanges(range.getEnd()));
```

Comment:
can there be a difference between `adjustInv(updatePositionDueToChanges(adjust(range.getEnd())))` and `updatePositionDueToChanges(range.getEnd())` assuming `adjust` moves one char right and `adjustInv` is the inverse?